trailblazer/server/api.py

```diff
@@ -186,10 +184,8 @@ def delete(analysis_id):
 def post_get_latest_analysis():
     """Return latest analysis entry for specified case id."""
     post_request: Response.json = request.json
```

Comment:
How come the `post_request` is a `Response.json` object?

Reply:
Dunno, should be Any or None. Removing it for now